jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -191,44 +197,67 @@ def _redshift_to_velocity(self, redshift):
         # the default case to avoid quantity math as below for efficiency
         return const.c.value * (zponesq - 1) / (zponesq + 1) / 1000  # km/s
 
+    def _update_line_positions(self):
+        # update all lines, self._global_redshift, and emit message back to Specviz helper
+        z = u.Quantity(self.rs_redshift)
```

Comment:
Here, you are invoking Quantity constructor on every update, even if `rs_redshift` has not changed. If performance is your goal, you might want to change it to Quantity somewhere else.